.github/workflows/ci.yaml

```diff
@@ -36,6 +36,14 @@ jobs:
         run: yarn dedupe --check
       - name: Build resources
         run: ./node_modules/.bin/gulp gen-icons-json build-translations build-locale-data gather-gallery-pages
+      - name: Setup lint cache
+        uses: actions/cache@v3.3.1
+        with:
+          path: |
+            node_modules/.cache/prettier
+            node_modules/.cache/eslint
+          key: lint-${{ github.sha }}
+          restore-keys: lint-
```

Comment:
This will restore any lint cache, even from other open PR (which doesn't match the description of the PR).

Reply:
PRs don't have access to each other's caches per the [security restrictions](https://docs.github.com/en/actions/using-workflows/caching-dependencies-to-speed-up-workflows#matching-a-cache-key).  The key is only searched in the current branch and then the default branch.

Reply:
TIL! Thanks! 👍